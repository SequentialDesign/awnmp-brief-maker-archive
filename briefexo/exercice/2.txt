Let's say you often do some typos, you're fed up of writing "thruogh" or whatever, and want your computer to fix it for you (I like when the computer do all the work for me :P)

Make a pre-dictionary on which:
- if you type "thruogh" or "trhough" or "throuhg" or "throu" or "thru" it returns "through" 
- if you type "cna" or "acn" return "can"
