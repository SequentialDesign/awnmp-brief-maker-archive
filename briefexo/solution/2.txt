thruogh, trhough, through, throu, thru : through |
cna, acn: can
