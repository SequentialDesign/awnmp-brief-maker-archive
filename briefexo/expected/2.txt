::thruogh::through
::trhough::through
::through::through
::throu::through
::thru::through
::cna::can
::acn::can
